From 21284a7cd17f56ba7894b47a6567cc84a0bf0509 Mon Sep 17 00:00:00 2001
From: Marek Vasut <marex@denx.de>
Date: Thu, 31 Aug 2023 16:41:15 +0200
Subject: [PATCH 2/7] ARM: imx: Use default SAVED_DRAM_TIMING_BASE on DH i.MX8M
 Plus DHCOM

Use default SAVED_DRAM_TIMING_BASE as that is what upstream TFA expects.
Without this change, the board will fail to suspend/resume e.g. in Linux.

Upstream-Status: Backport [bd7606d1421766f83474cf1539bd9dfaeb20cc5e]
Signed-off-by: Marek Vasut <marex@denx.de>
---
 configs/imx8mp_dhcom_pdk2_defconfig | 1 -
 configs/imx8mp_dhcom_pdk3_defconfig | 1 -
 2 files changed, 2 deletions(-)

diff --git a/configs/imx8mp_dhcom_pdk2_defconfig b/configs/imx8mp_dhcom_pdk2_defconfig
index 60ab31b6e8e..859e6dd1f03 100644
--- a/configs/imx8mp_dhcom_pdk2_defconfig
+++ b/configs/imx8mp_dhcom_pdk2_defconfig
@@ -161,7 +161,6 @@ CONFIG_SPL_CLK_COMPOSITE_CCF=y
 CONFIG_CLK_COMPOSITE_CCF=y
 CONFIG_SPL_CLK_IMX8MP=y
 CONFIG_CLK_IMX8MP=y
-CONFIG_SAVED_DRAM_TIMING_BASE=0x40000000
 CONFIG_DFU_TFTP=y
 CONFIG_DFU_TIMEOUT=y
 CONFIG_DFU_MMC=y
diff --git a/configs/imx8mp_dhcom_pdk3_defconfig b/configs/imx8mp_dhcom_pdk3_defconfig
index 6a98a906059..cb5572e081e 100644
--- a/configs/imx8mp_dhcom_pdk3_defconfig
+++ b/configs/imx8mp_dhcom_pdk3_defconfig
@@ -162,7 +162,6 @@ CONFIG_SPL_CLK_COMPOSITE_CCF=y
 CONFIG_CLK_COMPOSITE_CCF=y
 CONFIG_SPL_CLK_IMX8MP=y
 CONFIG_CLK_IMX8MP=y
-CONFIG_SAVED_DRAM_TIMING_BASE=0x40000000
 CONFIG_DFU_TFTP=y
 CONFIG_DFU_TIMEOUT=y
 CONFIG_DFU_MMC=y
-- 
2.42.0

